docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -64,8 +64,8 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 #### Q: should the number of retries of a job be limited, and if so, where? e.g. a max_retries field in the parent job? wait and see whether people attempt to rerun jobs that have already failed nine zillion times?
 A: Unknown TBD
 
-#### Q: Preventing the same params from being re-run
-A: We have decided to allow it
+#### Q: Preventing the identical params from being re-run within a retry_jobs request
+A: We have decided to allow multiple jobs with the same params to be re-run in the same retry_jobs request.
```

Comment:
when did we decide this? is it a temporary decision until the behaviour can be patched to eliminate duplicates?

Reply:
We decided this in our last engineering meeting

Reply:
@ialarmedalien I think the ADR already reflects this, so not sure what you mean. It says

`Multiple in-flight retries are allowed` as the current behavior and
`Prevent multiple in-flight retries of the same original job to prevent the user from wasting their own resources (and the queues resources)` is the desired behavior

Reply:
I would agree with you that it is a parameter, but my comment was  off of the models https://github.com/kbase/execution_engine2/blob/develop/lib/execution_engine2/db/models/models.py#L156 where you can see the Params are job params, and Source_ws_object is just a job input.

Reply:
OK -- I think this is all resolved, then. 🎉